dll/win32/setupapi/query.c

```diff
@@ -693,3 +693,21 @@ BOOL WINAPI SetupQueryInfOriginalFileInformationW(
 
     return TRUE;
 }
+
+BOOL WINAPI SetupQueryInfVersionInformationA(SP_INF_INFORMATION *info, UINT index, const char *key, char *buff,
+    DWORD size, DWORD *req_size)
+{
+    FIXME("info %p, index %d, key %s, buff %p, size %ld, req_size %p stub!\n", info, index, debugstr_a(key), buff,
+        size, req_size);
```

Comment:
This debug trace can be simplified a bit: parameter names like `info`, `index`, `key` etc. and the commas can be omitted in the format string. See the following example: https://github.com/reactos/reactos/blob/d1d300b44bee98c8ab11a68994710377e1a7f775/dll/win32/setupapi/stubs.c#L119
Also, (optionally) you can add a `__FUNCTION__` macro with `%s` identifier in the string, to display the current function name in debug print too.
```suggestion
    FIXME("%s(%p %d %s %p %ld %p) stub\n", __FUNCTION__, info, index, debugstr_a(key), buff, size, req_size);
```

Reply:
I agree with the `__FUNCTION__` comment (althought that could also be done by setting the trace format), but I don't think we actually improve this by omitting the argument names?
They make the log more readable without having to reference the code.